webapp/client/src/pages/RegistrationPage.js

```diff
@@ -33,7 +33,16 @@ export default function RegistrationPage({
   return (
     <>
       <StepHeaderButtons />
-      <FormHeader {...stepHeader} />
+      <FormHeader
+        {...stepHeader}
+        intro={
+          <Trans
+            t={t}
+            i18nKey="unlockCodeRequest.deadlineIntro"
+            components={{ bold: <strong /> }}
```

Comment:
I also just learned this a few weeks ago, but `<strong>` should not be used if you only want the text to appear bold.  See https://stackoverflow.com/a/271776 for reference. Instead, let's use `<b>`

Reply:
Changed it to <b>